protocols/ping/src/listen_handler.rs

```diff
@@ -21,21 +21,21 @@
 use arrayvec::ArrayVec;
 use futures::prelude::*;
 use libp2p_core::{
-    InboundUpgrade,
+    Upgrade,
     ProtocolsHandler,
     ProtocolsHandlerEvent,
     upgrade::DeniedUpgrade
 };
 use log::warn;
-use protocol::{Ping, PingListener};
+use protocol::{Pong, PingListener};
```

Comment:
Should it be `PongListener`? Pings are outbound, pongs are inbound right? So when listening we're getting `Pong`s?